tests/app/main/views/test_suppliers.py

```diff
@@ -309,7 +309,6 @@ def test_should_show_correct_fields_for_disabled_service(self, data_api_client):
 
         assert response.status_code == 200
         assert "Removed" in response.get_data(as_text=True)
-        assert "Details" in response.get_data(as_text=True)
```

Comment:
Should we be asserting that `Edit` is there instead?

Reply:
Maybe. But if we're going to do it properly then we should be rendering the document and using XPath lookup in the right place.  The test felt so frail as it was I didn't think losing this would make a lot of difference.

There, I've said it and now feel ashamed.

Reply:
😸 I guess revamping the tests to be stricter is one for the tech debt pile.

Reply:
I'm going to at least put the check for "Edit" in, as the following test for an enabled service has it.